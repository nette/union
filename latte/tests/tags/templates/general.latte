<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<input/> <input /> <input>

<input checked> <input checked="checked">

<button />

{ test} {"test} {'test}

{= (string) (bool) (float) (int) (array) 10}


{foreach array(true) as $foo}
	{foreach $people as $person}
	{if $iterator->isFirst()}<ul>{/if}
	<li id="item-{$iterator->getCounter()}" class="{$iterator->isOdd() ? 'odd' : 'even'}">{$person}</li>
	{if $iterator->isLast()}</ul>{/if}
	{/foreach}
{/foreach}


{var $counter = 0}
{block menu}
<ul>
	{foreach $menu as $item}
	<li>{$counter++} {if is_array($item)} {include this, 'menu' => $item} {else}{$item}{/if}</li>
	{/foreach}
</ul>
{/block}
