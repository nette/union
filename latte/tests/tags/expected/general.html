<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<input/> <input /> <input>

<input checked> <input checked="checked">

<button></button>

{ test} {"test} {'test}

1


	<ul>
	<li id="item-1" class="odd">John</li>


	<li id="item-2" class="even">Mary</li>


	<li id="item-3" class="odd">Paul</li>


	<li id="item-4" class="even">]]&gt; &lt;!--</li>
	</ul>


<ul>
	<li>0 about</li>
	<li>1  <ul>
	<li>2 product1</li>
	<li>3 product2</li>
</ul>
 </li>
	<li>2 contact</li>
</ul>
